app/DoctrineMigrations/Version20230321150513.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230321150513 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE restaurant ADD max_orders_range_duration INT DEFAULT NULL');
```

Comment:
Maybe we should name it `rate_limit_range_duration`/`rate_limit_amount`?

Reply:
Yep ! Totally agree, I changed the name of the feature in the middle of the day :laughing:

Reply:
Fixed with [a720d93](https://github.com/coopcycle/coopcycle-web/pull/3584/commits/a720d93b3820697dc439a03b28c24b7bda62b485)